dts/bindings/qspi/nxp,s32-qspi-sfp-mdad.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP S32 Quad Serial Peripheral Interface (QSPI) Secure Flash Protection SFP MDAD.
+
+  The SFP MDAD performs first-level checks on input transactions, based on the secure attribute
+  and MGID associated with each transaction.
+
+compatible: "nxp,s32-qspi-sfp-mdad"
+
+include: base.yaml
+
+properties:
+  secure-attribute:
+    type: string
+    enum:
+      - RESERVED
+      - UNSECURE
+      - SECURE
+      - BOTH
+    default: BOTH
+    description: |
+      Selects the secure attribute:
+      - RESERVED: This configuration should not be used
+      - UNSECURE: Allow the bus attribute for this master to non-secure only
+      - SECURE: Allow the bus attribute for this master to secure only
+      - BOTH: Allow the bus master's attribute both secure and non-secure
+
+  mask-type:
+    type: string
+    enum:
+      - MASK AND
+      - MASK OR
```

Comment:
we can drop the MASK prefix in all enums, already mentioned in the property

Reply:
I droped